openreview/conference/templates/bidWebfield.js

```diff
@@ -207,124 +211,19 @@ function renderContent(notes, conflictIds, bidEdges) {
     }
   });
 
-  function updateNotes(notes) {
-
-    var bidCount = 0;
-    $('#bidcount').remove();
-    $('#header').append('<h4 id="bidcount">You have completed ' + bidCount + ' bids</h4>');
-
-    var loadingContent = Handlebars.templates.spinner({ extraClasses: 'spinner-inline' });
-    sections = [
-      {
-        heading: 'All Papers  <span class="glyphicon glyphicon-search"></span>',
-        id: 'allPapers',
-        content: null
-      },
-      {
-        heading: 'Very High',
-        headingCount: bidsById['Very High'].length,
-        id: 'veryHigh',
-        content: loadingContent
-      },
-      {
-        heading: 'High',
-        headingCount: bidsById['High'].length,
-        id: 'high',
-        content: loadingContent
-      },
-      {
-        heading: 'Neutral',
-        headingCount: bidsById['Neutral'].length,
-        id: 'neutral',
-        content: loadingContent
-      },
-      {
-        heading: 'Low',
-        headingCount: bidsById['Low'].length,
-        id: 'low',
-        content: loadingContent
-      },
-      {
-        heading: 'Very Low',
-        headingCount: bidsById['Very Low'].length,
-        id: 'veryLow',
-        content: loadingContent
-      }
-    ];
-    sections[activeTab].active = true;
-
-    $('#notes .tabs-container').remove();
-
-    Webfield.ui.tabPanel(sections, {
-      container: '#notes',
-      hidden: true
+  $('#invitation-container').on('change', 'form.notes-search-form .score-dropdown', function(e) {
+    selectedScore = $(this).val();
+    return getPapersSortedByAffinity(0, selectedScore)
```

Comment:
passing selectedScore to getPapersSortedByAffinity is not necessary as the function takes only 1 argument which is the offset